### ISSUE TITLE
DEMOGRAPHICS INCIDENT PATIENTS

### ISSUE THREAD

**Issue by prubiom** [oxford-pharmacoepi/MegaStudy#75]
HI!

I see that some people here have had the same problem in this part oF DUS code, but I wasn't able to find the way to solve it. 

![image](https://github.com/user-attachments/assets/9f9c9b3d-5359-45c2-8a4c-f9d6a2a44bce)

**Comment by tiozab** [comment]
@prubiom please double check that the renv worked.

**Comment by prubiom** [comment]
![image](https://github.com/user-attachments/assets/df9095dc-d58b-4040-8589-02bfe0bf23fa)

**Comment by tiozab** [comment]
@prubiom yes, from these package version you can see that the renv has not worked.
Please make sure the renv has been activated and restored so that the correct package versions are used (as indicated in the renv lock file)
